Topics/Software_Engineering/Scrum.md

```diff
@@ -0,0 +1,68 @@
+- Development team:
+    - The development team is the ones who work on creating the product/working on items in the sprint, according to the specifications from the product owner. 
+    The team includes developers, UX specialists, Ops engineers, testers, and designers. 
+    With these differing skill sets, the team can cross-train each other to prevent bottlenecks.
+
+### Scrum artifacts
+Scrum artifacts refer to the information a scrum team uses that details information about the product in development, the tasks involved in a sprint cycle, and the end goal. 
+- Product backlog:
+    - The product backlog is the primary list of work that needs to be done and is maintained and updated by the product owner or manager.
+- Sprint backlog:
+    - The sprint backlog is the list of user stories or bug fixes that should be done by the end of the current sprint cycle and chosen from the product backlog.
+- Increment (sprint goal):
+    - The increment is the end product of a sprint. 
+    - The increment can mean a finished product, features usable to customers by the end of the sprint, or a completed section of a larger project.
+
+### Scrum events/ceremonies
+The scrum framework incorporates regular meetings and events that teams perform regularly. In scrum, there are five regularly held events:
+- Backlog organization:
+    - This is the responsibility of the product owner, who makes sure to continually update and maintain the product backlog, according to feedback from users and the development team.
+- Sprint planning:
+    - This meeting is led by the scrum master and includes the development team, where the items to be completed during the sprint are added from the product backlog per the sprint goal.
+- Sprint:
+    - This is the time period where the scrum team works to complete items in the scope of the sprint.
+- Daily standup:
+    - The standup is a regularly scheduled meeting in which members of the team will update members on their progress and mention blockers they are facing with their work.
+- Sprint review:
+    - This occurs at the end of the sprint, where the team meets to demo the end product and showcase the completed sprint backlog items.
+- Sprint retrospective: 
+    - Also occurring at the end of the sprint, the retro is where the team discusses the aspects of the sprint that worked and parts that could use improvement.
+    - This builds in feedback and continual improvement of processes in the scrum framework.
+
+### Why is scrum important?
+Teams use the scrum framework since it provides an efficient and adaptable way to organize and manage teams and products. It is team-centric and self-managed and encourages creativity with the flexibility to assign work based on work styles. The framework has concrete roles, events, artifacts, and values. These aspects of scrum are incorporated into professional workplaces and can be used in the 301 to finish the project in the short amount of time given. 
```

Comment:
Minor grammar thing, I would replace "the 301" with "CSC301"/"this course".

Reply:
Thanks for that catch!